### PR TITLE
Controls.Hosting(GTK): mappings for Frame

### PR DESCRIPTION
**Pull request by webwarrior-ws**
Add mappers for BorderColor and CornerRadius for Frame elements so they can now have borders.

### CODE REVIEW

**Comment by knocte** [comment]
@webwarrior-ws can you resolve the conflicts please?

**Comment by webwarrior-ws** [comment]
> @webwarrior-ws can you rebase, to resolve the conflicts please?

This PR points to branch in @aarani 's repo, so I can't rebase. I only can resolve conflicts using Github UI.

**Comment by aarani** [comment]
@jsuarezruiz Is there anything else that needs to be done  here?

**Comment by knocte** [comment]
@jsuarezruiz can you review please? I know we can always work with our fork but we prefer to use your repo as "upstream".

**Comment by knocte** [comment]
@jsuarezruiz ping

**Comment by lytico** [comment]
[ @knocte wrote:](https://github.com/jsuarezruiz/maui-linux/pull/66#issuecomment-2011800576)

> BTW by any chance can you review https://github.com/jsuarezruiz/maui-linux/pull/51 ?

i have seen ist, but i'm not happy with this solution.

to my opinion this should be done in FrameHandler.Gtk (src/Controls/src/Core/Compatibility/Handlers/Gtk/FrameHandler.Gtk.cs) and done via Map / Update-Pattern

i made a refactoring here:
https://github.com/lytico/maui/pull/288

File: src/Controls/src/Xaml/Hosting/AppHostBuilderExtensions.cs [hunk 1]
```diff
@@ -249,5 +251,34 @@ internal static MauiAppBuilder RemapForControls(this MauiAppBuilder builder)
 
 			return builder;
 		}
+
+		internal static void AddDefaultMappings()
+		{
+#if GTK
+			ContentViewHandler.Mapper.AppendToMapping(
```
Comment (jsuarezruiz) [inline src/Controls/src/Xaml/Hosting/AppHostBuilderExtensions.cs:258]: I think will be better to manage and maintain to move this to FrameHandler.Gtk

Comment (lytico) [reply src/Controls/src/Xaml/Hosting/AppHostBuilderExtensions.cs:258]: yes, this is already done in https://github.com/lytico/maui/pull/289 and part of [lytico/merge/main240321](https://github.com/lytico/maui/tree/lytico/merge/main240321)

**Pull request closed without merging**